db/schema.rb

```diff
@@ -765,16 +785,6 @@
     t.index ["account_uuid"], name: "tud_account_availability_logs"
   end
 
-  create_table "test_user_dashboard_tud_account_checkouts", force: :cascade do |t|
-    t.string "account_uuid"
-    t.datetime "checkout_time"
-    t.datetime "checkin_time"
-    t.boolean "has_checkin_error"
-    t.boolean "is_manual_checkin"
-    t.datetime "created_at", precision: 6, null: false
-    t.datetime "updated_at", precision: 6, null: false
-  end
```

Comment:
Should this table be removed?

Reply:
Oh boy, definitely not. I think I may have something weird locally going on.

Reply:
Yeah, I've definitely had that happen before too 😄

Reply:
Ok, the schema oopsie accident is updated 🤣